examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_2.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
Please, provide a table with perf comparison of the operations used here (pandas vs Modin on OmniSci). You can put it at the PR description section.

Reply:
@YarShev I added this table